includes/controllers/class.llms.controller.certificates.php

```diff
@@ -117,10 +117,33 @@ public function maybe_handle_reporting_actions() {
 			$this->download( $cert_id );
 		} elseif ( isset( $_POST['llms_delete_cert'] ) ) {
 			$this->delete( $cert_id );
+		} elseif ( isset( $_POST['llms_enable_cert_sharing'] ) || isset( $_POST['llms_disable_cert_sharing'] ) ) {
+			$this->change_sharing_settings( $cert_id, isset( $_POST['llms_enable_cert_sharing'] ) );
```

Comment:
```suggestion
		} elseif ( isset( $_POST['llms_enable_cert_sharing'] ) ) {
			$this->change_sharing_settings( $cert_id, (bool) $_POST['llms_enable_cert_sharing'] );
		}
```
Since `<button>`s can have a value why don't we use one var only?